pkg/test/xds/client/Makefile

```diff
@@ -1,5 +1,6 @@
-NUM_OF_DATAPLANES ?= 10
+NUM_OF_DATAPLANES ?= 100
+NUM_OF_SERVICES ?= 80
 KUMA_CP_ADDRESS ?= grpcs://localhost:5678
 
 run:
-	go run main.go run --total "${NUM_OF_DATAPLANES}" --xds-server-address "${KUMA_CP_ADDRESS}"
\ No newline at end of file
+	go run main.go run --dps "${NUM_OF_DATAPLANES}" --services "${NUM_OF_SERVICES}" --xds-server-address "${KUMA_CP_ADDRESS}"
```

Comment:
Can we get rid of this makefile and move it `mk/run.mk` ? Also maybe this whole package should be moved to `tools/` that's where we usually put these things now.